string robot_id
time stamp
uint16 system_dim
float64[] mean_vector
float64[] info_vector
utils/SparseMatrix info_matrix